pkg/apis/feeds/v1alpha1/feed_types.go

```diff
@@ -52,17 +52,8 @@ type FeedSpec struct {
 // FeedAction specifies the target handler - a Knative route or channel - for
 // events produced by an EventTrigger.
 type FeedAction struct {
-	// You must specify one and only of these.
-
-	// RouteName specifies Knative route as a target.
-	RouteName string `json:"routeName,omitempty"`
-
-	// ChannelName specifies the channel name as a target
-	// If ChannelName specifies a full DNS (for example:
-	// flow-example-channel.default.svc.cluster.local)
-	// it's returned as is.
-	// TODO: clean up the action names.
-	ChannelName string `json:"channelName,omitempty"`
+	// DNSName is the target for this feed.
+	DNSName string `json:"dnsName,omitempty"`
```

Comment:
I have a feeling there will still be some other changes to CRDs, so I think I agree with @evankanderson that for now it might be better to avoid churn.